plugins/field-bitmap/src/field-bitmap.ts

```diff
@@ -0,0 +1,595 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview Bitmap input field.
+ * @author gregoryc@outlook.com (Greg Cannon)
+ */
+
+import Blockly from 'blockly/core';
+
+export const DEFAULT_HEIGHT = 5;
```

Comment:
note on the lint errors: rachel has an update to the eslint config that allows constant case. it's bundled in another pr. I will poke her about submitting that when she is feeling better

Reply:
Ah nice, and sounds good!